### PR TITLE
[16.0][IMP] rma: dedicated settings section

### PR DESCRIPTION
**Pull request by sbejaoui**
image

### CODE REVIEW

File: rma/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -2,21 +2,30 @@
 <odoo>
     <record id="res_config_settings_view_form" model="ir.ui.view">
         <field name="model">res.config.settings</field>
-        <field name="inherit_id" ref="stock.res_config_settings_view_form" />
+            <field name="inherit_id" ref="base.res_config_settings_view_form" />
```
Comment (chienandalu) [inline rma/views/res_config_settings_views.xml]: Maybe we could avoid some of these diff spaces

**Review by chienandalu — APPROVED** [review]
Nice! :+1:

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by chienandalu — APPROVED** [review]
Great now! :+1:

**Review by pedrobaeza — APPROVED** [review]
The only ugly thing is to have the sample screenshot with enterprise installed, hehe.

/ocabot merge minor

**Pull request merged**